storybookjs/storybook
# Add absolute url for screenshot to show up on npm

Issue: Screenshot does not show up on npm

## What I did

Add absolute path to the image on github so it will show up on npm 

## How to test

- Is this testable with Jest or Chromatic screenshots?
- Does this need a new example in the kitchen sink apps?
- Does this need an update to the documentation?

If your answer is yes to any of these, please make sure to include it in your PR.

Issue
<img width="1118" alt="@storybook:addon-info - npm 2019-04-12 03-20-20" src="https://user-images.githubusercontent.com/12564956/56030547-f5024e00-5cd1-11e9-856c-25232af9c3f9.png">
